### ISSUE TITLE
Issue w/ rownames when bulk RNAseq data is only one sample

### ISSUE THREAD

**Issue by jr-leary7** [amitfrish/scBio#12]
You need to change the code when creating the `YReduced` object in the `CPMMain()` function from:

```
YReduced = Y[row.names(Y) %in% row.names(refference), ]
```

to:

```
YReduced = Y[row.names(Y) %in% row.names(refference), , drop = FALSE]
```

This will prevent rownames being dropped from `YReduced`, which causes problems in the next line.

**Comment by amitfrish** [comment]
Sorry for the delay in my response.
I'm checking it now and if I'll update the CRAN version as soon as I fix it.
Thank you for your insight.

**Comment by amitfrish** [comment]
I fixed the bug and updated to version 0.1.6
